pkg/materialize/cluster.go

```diff
@@ -151,7 +165,9 @@ func (b *ClusterBuilder) Create() error {
 
 		if b.schedulingConfig.OnRefresh.Enabled {
 			scheduleStatement := " SCHEDULE = ON REFRESH"
-			if b.schedulingConfig.OnRefresh.RehydrationTimeEstimate != "" {
+			if b.schedulingConfig.OnRefresh.HydrationTimeEstimate != "" {
+				scheduleStatement += fmt.Sprintf(" (HYDRATION TIME ESTIMATE = %s)", QuoteString(b.schedulingConfig.OnRefresh.HydrationTimeEstimate))
+			} else if b.schedulingConfig.OnRefresh.RehydrationTimeEstimate != "" {
 				scheduleStatement += fmt.Sprintf(" (REHYDRATION TIME ESTIMATE = %s)", QuoteString(b.schedulingConfig.OnRefresh.RehydrationTimeEstimate))
```

Comment:
Would it be possible to make Terraform write "HYDRATION TIME ESTIMATE" in the SQL, even when the user used the `RehydrationTimeEstimate` option at the Terraform level? This way, even if some Terraform users don't move to the new option name at the Terraform level, we can still remove the old syntax from SQL, as soon as they all simply move to the new version of Terraform.

Reply:
Good idea! Will change that now!